pkg/governance-scripts/contracts/BalancerContractRegistryInitializer.sol

```diff
@@ -78,17 +77,6 @@ contract BalancerContractRegistryInitializer {
 
         _initialized = true;
 
-        // Grant permissions to register contracts and add aliases.
-        bytes32 registerContractRole = IAuthentication(address(balancerContractRegistry)).getActionId(
-            IBalancerContractRegistry.registerBalancerContract.selector
-        );
-        bytes32 addAliasRole = IAuthentication(address(balancerContractRegistry)).getActionId(
-            IBalancerContractRegistry.addOrUpdateBalancerContractAlias.selector
-        );
-
-        _authorizer.grantRole(registerContractRole, address(this));
-        _authorizer.grantRole(addAliasRole, address(this));
```

Comment:
We start with the assumption that the permissions have already been granted. If they are not, the script will fail with `SENDER_NOT_ALLOWED`.

That's kinda fine; another option would be to check that the script effectively has the permissions before executing them and reverting with a custom error. I don't have a strong opinion.

Reply:
If we're adopting this as a new pattern, a custom error isn't a bad idea.